internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestVeniceHelixAdminWithSharedEnvironment.java

```diff
@@ -2054,4 +2054,34 @@ public void testRaceConditionFixForKillOfflinePushAndVersionSwap(boolean isKillO
     stopParticipant(newNodeId);
   }
 
+  @Test
+  public void testInstanceTagging() {
+    List<String> instanceTagList = Arrays.asList("GENERAL", "TEST");
+    String controllerClusterName = "venice-controllers";
+    int newAdminPort = controllerConfig.getAdminPort() + 1; /* Note: dummy port */
+    PropertyBuilder builder = new PropertyBuilder().put(controllerProps.toProperties()).put("admin.port", newAdminPort);
+
+    VeniceProperties newControllerProps = builder.build();
+    VeniceControllerClusterConfig newConfig = new VeniceControllerClusterConfig(newControllerProps);
+    VeniceHelixAdmin newHelixAdmin = new VeniceHelixAdmin(
```

Comment:
Since we've set the config in `AbstractTestVeniceHelixAdmin`, we shouldn't need to create a new `VeniceHelixAdmin` object